.circleci/config.yml

```diff
@@ -67,90 +134,105 @@ jobs:
           command: .circleci/tag_and_push_docker_image \
               nginx:$CIRCLE_SHA1 \
               ${ECR_DOCKER_REPO_BASE}-nginx
-  lint-web:
+
+  live-0_staging_deploy:
     docker:
-      - image: circleci/python:3.7
+      - image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci
     steps:
       - checkout
       - run:
-          name: Setup Python environment
+          name: Initialise Kubernetes staging context
           command: |
-            pip install virtualenv
-            virtualenv lint-env
-            source lint-env/bin/activate
-            pip install pip==18.1
-      - restore_cache:
-          keys:
-            - pip-v1-{{ checksum "requirements/lint.txt" }}
-      - run:
-          name: Install dependencies
+            setup-kube-auth
+            kubectl config use-context staging
+      - deploy:
+          name: Deploy fala to staging
           command: |
-            source lint-env/bin/activate
-            pip install --requirement requirements/lint.txt
-      - save_cache:
-          key: pip-v1-{{ checksum "requirements/lint.txt" }}
-          paths:
-            - "~/.cache/pip"
+            .circleci/deploy_to_kubernetes_live_0 staging_live_0 ${ECR_DOCKER_REPO_BASE}-webapp ${ECR_DOCKER_REPO_BASE}-nginx
+      - deploy:
+          name: Notify Slack channel
+          command: .circleci/notify_slack_channel staging-live-0
+
+  live-0_production_deploy:
+    docker:
+      - image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci
+    steps:
+      - checkout
       - run:
-          name: Lint with flake8
+          name: Initialise Kubernetes production context
           command: |
-            source lint-env/bin/activate
-            flake8 fala
-      - run:
-          name: Check formatting with Black
+            setup-kube-auth
+            kubectl config use-context production
+      - deploy:
+          name: Deploy fala to production
           command: |
-            source lint-env/bin/activate
-            black --check fala
-  test-web:
+            .circleci/deploy_to_kubernetes_live_0 production ${ECR_DOCKER_REPO_BASE}-webapp ${ECR_DOCKER_REPO_BASE}-nginx 
+      - deploy:
+          name: Notify Slack channel
+          command: .circleci/notify_slack_channel production-live-0
+
+  live-1_build:
     docker:
-      - image: circleci/python:3.4
+      - image: docker:17.03-git
+    environment:
+      DSD_DOCKER_REGISTRY: "registry.service.dsd.io"
+      DSD_DOCKER_IMAGE: "fala"
     steps:
       - checkout
+      - setup_remote_docker:
+          version: 17.03.0-ce
+          docker_layer_caching: true
       - run:
-          name: Setup Python environment
+          name: Login to the DSD Docker registry
```

Comment:
_not a blocker_

🙋‍♂ Probably not for now, but we can get rid of all DSD (template-deploy) remnants. It also means we can upgrade Docker to 18.09. 🎉